server/config.go

```diff
@@ -56,6 +56,10 @@ type LeveldbOptions struct {
 	BloomKeyBits        int
 	CompactionTableSize int
 	CompactionTotalSize int
+
+	CompactionTableSizeMultiplier float64
+
+	DisbableCompression bool
```

Comment:
After `disable`, all `Compaction***` would not work?

Reply:
no, this is leveldb snappy compression(data level), not related to `Compaction***` (key-value level)